src/components/Transaction/ParameterList.vue

```diff
@@ -0,0 +1,80 @@
+<script>
+import JsonViewer from 'vue-json-viewer';
+import AddressField from 'components/AddressField';
+export default {
+    name: 'ParameterList',
+    components: {
+        JsonViewer,
+        AddressField,
+    },
+    methods: {
+        expand(param, value) {
+            this.expanded[param][value] = true;
+        },
+        minimize(param, value) {
+            this.expanded[param][value] = false;
+        },
+    },
+    data(){

```

Comment:
can be simplified to
```js
data: () => ({
    expanded: Array(this.params.length).fill([])
})
```

Reply:
This actually doesn't work, for some reason when we call toggle (ie: this.expanded[index]['expanded'] = !this.expanded....) next it changes the whole array